packages/core/test/scxml.test.ts

```diff
@@ -230,13 +230,13 @@ const testGroups = {
     'test335.txml',
     'test336.txml',
     'test337.txml',
-    // 'test338.txml', // _event.invokeid not implemented yet
+    'test338.txml',
```

Comment:
This is passing only by accident - because both variables that are being compared yield `undefined` value but to make this test actually working both should be assigned with correct values
```suggestion
    // 'test338.txml', // <invoke idlocation="..."> + _event.invokeid available on <send> events received from the invoked child
```